config/initializers/health_monitor.rb

```diff
@@ -3,15 +3,20 @@
   HealthMonitor.configure do |config|
     config.cache
 
+    config.file_absence.configure do |file_config|
+      file_config.filename = "public/remove-from-nginx"
+    end
     # Mediaflux check
     config.add_custom_provider(MediafluxStatus)
 
     # Make this health check available at /health
     config.path = :health
 
     config.error_callback = proc do |e|
-      Rails.logger.error "Health check failed with: #{e.message}"
-      Honeybadger.notify(e)
+      unless e.is_a?(HealthMonitor::Providers::FileAbsenceException)
```

Comment:
Same as Geaccirc - It occurred to me after I did this on another app that we probably do still want it to log to the Rails logger, in case we're like "why the heck is this not on the load balancer?"